deployment/roles/oracle/tasks/jumpbox.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: Build the containers
-  shell: docker-compose build
+- name: Pull the containers images
+  shell: docker-compose pull
```

Comment:
Please double check. Most probably you need to remove the container explicitly here:
https://github.com/poanetwork/tokenbridge/blob/49291974d63068aa5eb1329d11862b53e4527955/deployment/roles/oracle/tasks/post_config.yml#L4
here
https://github.com/poanetwork/tokenbridge/blob/49291974d63068aa5eb1329d11862b53e4527955/deployment/roles/oracle/tasks/post_config.yml#L21
and here
https://github.com/poanetwork/tokenbridge/blob/49291974d63068aa5eb1329d11862b53e4527955/deployment/roles/oracle/tasks/post_config.yml#L32

otherwise you will get them exited but not deleted after the deployment process finish:
```
abf7dbb1e5ae        poanetwork/tokenbridge-oracle:latest   "node scripts/initia…"   43 seconds ago       Exited (0) 24 seconds ago                       oracle_bridge_affirmation_run_43b336ccf3ed
b875ee449858        poanetwork/tokenbridge-oracle:latest   "node scripts/privat…"   About a minute ago   Exited (0) 44 seconds ago                       oracle_bridge_affirmation_run_a23f8fbc6a45
```

Reply:
Good catch! I was able to reproduce the issue and fix it https://github.com/poanetwork/tokenbridge/pull/280/commits/345365b8f43e3c6cb2c58076a876a3271ee7f7bf